js/src/general/YoutubePlayer.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+
+export const YoutubePlayer = ({vid, ...props}) => (
+    <div {...props}>
+        <div className={`video-wrapper youtube mt-3 mb-4`} style={aspectRatio_16_9_Style} >
```

Comment:
```suggestion
        <div className="video-wrapper youtube mt-3 mb-4" style={aspectRatio_16_9_Style} >
```

regarding `aspectRatio_16_9_Style` I've tried to put all styles in css unless they have to be truly dynamic, probably best to follow that convention.

Reply:
Got it.